lib/App/Pinto/Command/look.pm

```diff
@@ -0,0 +1,85 @@
+# ABSTRACT: unpack and open distributions with your shell
+
+package App::Pinto::Command::look;
+
+use strict;
+use warnings;
+
+#------------------------------------------------------------------------------
+
+use base 'App::Pinto::Command';
+
+#------------------------------------------------------------------------------
+
+# VERSION
+
+#-----------------------------------------------------------------------------
+
+sub opt_spec {
+    my ( $self, $app ) = @_;
+
+    return (
+        [ 'shell=s' => 'The path to the shell command to use' ],
+    );
+}
+
+#------------------------------------------------------------------------------
+
+sub validate_args {
+    my ( $self, $opts, $args ) = @_;
+
+    $self->usage_error('Requires at least one target')
+        unless @{$args};
+
+    return 1;
+}
+
+sub args_attribute { return 'targets' }
+
+#------------------------------------------------------------------------------
+
+sub args_from_stdin { return 1 }
+
+#------------------------------------------------------------------------------
+
+1;
+
+__END__
+
+
+=head1 SYNOPSIS
+
+  pinto --root=REPOSITORY_ROOT look [OPTIONS] TARGET ...
+
+=head1 DESCRIPTION
+
+Unpack one or more distributions and open directories with your shell.
+Handy if you want to manually review a distribution before use.
+
+=head1 COMMAND ARGUMENTS
+
+Arguments are the targets you wish to unpin.  Targets can be
```

Comment:
he he. nice catch.


Kahlil (Kal) Hodgson                       GPG: C9A02289
Head of Technology                         (m) +61 (0) 4 2573 0382
DealMax Pty Ltd

Suite 1416
401 Docklands Drive
Docklands VIC 3008 Australia

"All parts should go together without forcing.  You must remember that
the parts you are reassembling were disassembled by you.  Therefore,
if you can't get them together again, there must be a reason.  By all
means, do not use a hammer."  -- IBM maintenance manual, 1925

On 2 March 2015 at 08:12, Jeffrey Ryan Thalhammer <notifications@github.com>
wrote:

> In lib/App/Pinto/Command/look.pm
> <https://github.com/thaljef/Pinto/pull/184#discussion_r25571513>:
>
> > +
> > +__END__
> > +
> > +
> > +=head1 SYNOPSIS
> > +
> > +  pinto --root=REPOSITORY_ROOT look [OPTIONS] TARGET ...
> > +
> > +=head1 DESCRIPTION
> > +
> > +Unpack one or more distributions and open directories with your shell.
> > +Handy if you want to manually review a distribution before use.
> > +
> > +=head1 COMMAND ARGUMENTS
> > +
> > +Arguments are the targets you wish to unpin.  Targets can be
>
> "unpin" -> "look at"
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/thaljef/Pinto/pull/184/files#r25571513>.
>